webapp/src/redux/features/plugins/PluginsState.ts

```diff
@@ -13,12 +13,14 @@ export const enum BuiltInPlugins {
     MsGraph = 'Microsoft Graph',
     Jira = 'Jira',
     GitHub = 'GitHub',
+    APIConnector = 'API Connector',
```

Comment:
Some comment here, change this to something like MsGraphObo

(This is not the proper use of the keyword `connector` in this case, since you're enabling a plugin auth flow)

Reply:
done - changed all the readme.md and references